tests/test_wave_05.py

```diff
@@ -80,34 +78,34 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
-    # Act
-    # ---- Complete Act Here ----
-
-    # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
-
+    response = client.put("/goals/1", json={
+        "title": "Build good habits"})
+    response_body = response.get_json()
+    
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "Build good habits"
+        }
+    }
```

Comment:
Note that it is redundant to have both of these asserts since the second assert covers the first one. I'd recommend only having the second assert (below) since it is more specific.
```
assert response_body == {
    "goal": {
        "id": 1,
        "title": "Updated Goal Title"
    }
}
```